locallib.php

```diff
@@ -58,4 +58,82 @@ function install_and_remove_block() {
             set_config("active_feedback", 0, "block_coursefeedback");
         }
     }
+}
+
+/**
+ * Returns the coursefeedback results for all courses and a given feedback and question
+ * Returns only the results for a specific course if $specificcourseid is provided
+ *
+ * @param int $questionid
+ * @param int $coursefeedbackid
+ * @param int $answerlimit
+ * @param int $showperpage
+ * @param int $page
+ * @return array of objects
```

Comment:
Hier würde ich unbedingt den Type der Objekte beschreiben, die in dem Rückgabe-Array sind. Also was für Felder die definitiv haben werden. (Ja, man kann auch einfach die SQL Query angucken, aber soll man ja nicht müssen, ne. :wink: 

Ansonsten, wenn man schon die Funktion Dokumentiert, würde ich auch ordentlich ein paar Worte zu allen Parametern schreiben. (Es fehlt auch in der `@param` Liste jetzt `$specificcourseid`.)

Ist jetzt in der ganzen Codebasis sowieso nicht so konsequent, aber wie gesagt, wenn schon, denn schon.

Reply:
Ergänzt in: 304c1193e3f45081b939070b1d0a549ff93baa91